feature-pack/src/main/resources/content/bin/wsprovide.bat

```diff
@@ -3,6 +3,8 @@
 @if not "%ECHO%" == ""  echo %ECHO%
 @if "%OS%" == "Windows_NT" setlocal
 
+set SERVER_OPTS=%*
```

Comment:
Same comment about maybe needing to be quoted. I know Windows can be picky about spaces which is why I think we generally quote the sets.